src/dewey/curation.clj

```diff
@@ -313,7 +313,7 @@
 
```

Comment:
I'm thinking about how to go about documenting our events not at the service level, but at the architecture level.
The `dewey.curation.resolve-consumer` function has the list of routing keys used by `dewey`, just as the `info-typer.messaging.routing-functions` function contains routing keys used by `info-typer`.

For now, this is fine. But, from my experience, tracking this stuff by diving into each service can be quite laborious. We should consider ways of centralizing the routing keys in a way that makes them "traceable", somehow.